extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -283,8 +283,13 @@ class AmpAccordion extends AMP.BaseElement {
     // overidden.
     const target = dev().assertElement(event.target);
     const header = dev().assertElement(event.currentTarget);
-    const anchor = closest(target, e => e.tagName == 'A', header);
-    if (anchor === null) {
+    const anchorOrTapTarget = closest(target, e => {
+      return (e.tagName == 'A')
+        || (e.hasAttribute('on')
+          && e.getAttribute('on')./*OK*/matches(/(^|;)\s*tap\s*/));
```

Comment:
You wouldn't have to do a null check if you cached the RegEx outside the class and called `exec`. Creating the RegEx outside the method saves time since the function doesn't have to reconstruct the RegEx every time it's called.

```js
const ON_TAP_RE = /(^|;)\s*tap\s*/g;
/*...*/
if (e.tagName == 'A' && ON_TAP_RE.exec(e.getAttribute('on')) { 
  /*...*/
}
```

Reply:
Not sure this RegEx is very rigorous though. Consider an element with id `tape`: `on="input-debounced:tape.show"`.